docs/source/environments/braintumor.rst

```diff
@@ -0,0 +1,163 @@
+
+- :math:`\bar{x} = x / L`
+- :math:`\bar{t} = \rho \cdot t`
+- :math:`\bar{c} = (cL^3) / c_0`
+
+where L represents the length of the one-dimensional spatial domain, :math:`c_0` represents the initial number of tumor cells and works as an arbitrary scaling term. This yields the following PDE:
+
+.. math::
+
+  \begin{aligned}
+    \bar{c}_t(\bar{x}, \bar{t}) &= D^* \nabla^2 \bar{c} +  \bar{c} - R^*(\alpha, \beta, d(\bar{x}, \bar{t}), \bar{t}) \bar{c}\,,  \quad && (\bar{x}, \bar{t}) \in [0, 1] \times [0, \bar{T}] \\
+    \bar{c}(\bar{x}, 0) &= L^3 e^{-100 \bar{x}^2}\,, \quad && \bar{x} \in [0, 1]  \\ 
+  \end{aligned}
+
+where quantities :math:`D^*` and :math:`R^*` are given by:
+
+.. math::
+   D^* = D/(\rho L^2)
+
+.. math::
+   R^*(\alpha, \beta,  d(\bar{x}, \bar{t}), \bar{t}) =
+   \begin{cases}
+   (1 - S(\alpha, \beta, d(\bar{x}, \bar{t}))) / \rho, & \text{if } \bar{t} \in \bar{\mathcal{T}}_\text{therapy} \subset \bar{\mathcal{T}} \\ 
+   0, & \text{otherwise}
+   \end{cases}
+
+
+Model Implementation Details
+------------------------
+
+Glioblastoma diagnosis and treatment monitoring are typically performed using two types of MRI scans: gadolinium enhanced T1-weighted and T2-weighted imaging (referred to here as T1 and T2, respectively). While these scans do not directly measure tumor cell density, modeling literature based on heuristic thresholds infer that [5]_:
+
+- The T1 region corresponds to areas of high tumor cell density, typically >80% of the carrying capacity
+- The T2 region corresponds to areas of moderate tumor cell density, typically >16% of the carrying capacity
+
+Using these thresholds, our model is capable of generating virtual MRI predictions allowing us to track T1 and T2 tumor radii dynamically over time and spatially define dose regions during radiation therapy administration.
+
+To simulate the full treatment cycle–diagnosis, therapy, and follow-up–we allow the virtual tumor to grow until the T1 region reaches a radius of 10-20mm, triggering the onset of therapy. We then apply radiation according to a predefined schedule, followed by a post-treatment simulation period of 100 days of free growth.
+
+
+Numerical Implementation
+------------------------
+
+
+The external beam radiation therapy schedule follows the standard protocol adopted by the University of Washington Medical Center, and is divided into two sequential stages:
+
+- Stage 1: 28 days of 1.8 :math:`(Gy)` each, delivered to the T2 region + a 25 mm margin
+- Stage 2: 6 days of 1.8 :math:`(Gy)` each, delivered to the T1 region + a 20 mm margin
+
+This results in a total dose of 61.2 :math:`(Gy)` delivered over 34 days of treatment.
+Radiation is administered on a 5-days-on, 2-days-off schedule to account for weekend breaks.
+
+
+Nondimensional Model
+------------------------
```

Comment:
Need to descibe you notation of c_j^n = c(x_j, t_n) or something. Perhaps I forgot to do this in my write up as well. If, so would be awesome if you can add it.